src/main/webapp/scripts/cookies.js

```diff
@@ -0,0 +1,84 @@
+const HOURS_PER_DAY = 24;
+const MINUTES_PER_HOUR = 60;
+const SECONDS_PER_MINUTE = 60;
+const SECONDS_PER_DAY = HOURS_PER_DAY * MINUTES_PER_HOUR * SECONDS_PER_MINUTE;
+const MILLISECONDS_PER_SECOND = 1000;
+
+/**
+ * Get the value of a cookie.
+ */
+function getCookie(name) {
+  let decodedCookie = decodeURIComponent(document.cookie);
+  let cookies = decodedCookie.split(';');
+  for (let i = 0; i < cookies.length; i++) {
+    cookieParts = cookies[i].trim().split("=");
+    if (cookieParts[0] === name) {
+      return cookieParts[1];
+    }
+  }
+  return "";
+}
+
+/**
+ * Create a new cookie based on name, value and days before expiring.
+ */
+function setCookie(name, value, expirationDays) {
+  let date = new Date();
+  date.setTime(date.getTime+(expirationDays*SECONDS_PER_DAY*MILLISECONDS_PER_SECOND));
```

Comment:
[Nit] Same note here about spaces around the binary operators.  

Also, I would personally remove the parens around the multiplication since it is well known that multiplication happens before addition in term of order of operation.  However, you may keep them if you think aid readability.